plugin-build/src/main/kotlin/io/sentry/android/gradle/transforms/MetaInfStripTransform.kt

```diff
@@ -63,6 +64,13 @@ abstract class MetaInfStripTransform : TransformAction<MetaInfStripTransform.Par
                         continue
                     }
 
+                    if (jarEntry.isSignatureEntry) {
+                        logger.warning("Signed Multirelease Jar (${jarFile.name}) found, skipping transform. This might lead to duplicate class errors due to a bug in AGP (https://issuetracker.google.com/issues/206655905). Please update to AGP >= 7.1.2 (https://developer.android.com/studio/releases/gradle-plugin)")
```

Comment:
```suggestion
                        SentryPlugin.logger..warning { "Signed Multirelease Jar (${jarFile.name}) found, skipping transform. This might lead to auto-instrumentation issues due to a bug in AGP (https://issuetracker.google.com/issues/206655905). Please update to AGP >= 7.1.2 (https://developer.android.com/studio/releases/gradle-plugin) in order to keep using `autoInstrumentation` option." }
```

* Let's use the logger from `SentryPlugin` to not create new instances.
* Let's also use a `warning {}` extension, that would prepend `[sentry]` to each log call
* I rephrased a bit the message